README.md

```diff
@@ -348,7 +348,43 @@ if (!sessionClient.isAuthenticated()) {
 
 ```
 
-**Note**: To get a **sessionToken**, you must use [Okta's Authentication API](https://developer.okta.com/docs/api/resources/authn/#application-types). You can use [Okta Java Authentication SDK](https://github.com/okta/okta-auth-java) to get a [sessionToken][session-token]. An example of using the Authentication API can be found [here](https://github.com/okta/samples-android/tree/master/custom-sign-in). The Authentication SDK is only available for API 24 and above. If using API < 24, we recommend using [chrome custom tabs][[chrome-custom-tabs]] but if you must implement a native UI then we've provided a set of [authn-android](https://github.com/okta/okta-oidc-android/tree/authn_android) libraries built using bazel's desugar tool.
+### Okta Java Authentication Setup
+
+To get a **sessionToken**, you must use [Okta's Authentication API](https://developer.okta.com/docs/api/resources/authn/#application-types). You can use [Okta Java Authentication SDK](https://github.com/okta/okta-auth-java) to get a [sessionToken][session-token]. An example of using the Authentication API can be found [here](https://github.com/okta/samples-android/tree/master/sign-in-kotlin). The Authentication SDK is only available for API 24 and above. If using API < 24, you must use Android Studio 4.0 or higher with [Java 8+ API desugaring][java-8-api].
```

Comment:
I think it's better to use https://developer.okta.com/docs/reference/api/authn/ as the link to `Okta's Authentication API` instead of https://developer.okta.com/docs/api/resources/authn/#application-types